### PR TITLE
Merge profile

### PR DESCRIPTION
**Pull request by Connor-Quinny**
-Created profile form
-Profile data can send to firebase

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -10,16 +12,20 @@ const Stack = createNativeStackNavigator();
 
 export default function App() {
 
-
```
Comment (MartinSWDev) [inline App.js:13]: can we sort these spaces out

File: src/components/CreateProfile.jsx [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+  postcode: "",
+  dancestyles: "",
+  role: "",
+  range: 0,
+  available: false,
+  about: ""
+}
+
+// const location = 'manchester';
+// const danceStyle = 'salsa';
```
Comment (MartinSWDev) [inline src/components/CreateProfile.jsx:31]: these comments need removing

File: src/components/CreateProfile.jsx [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+  const onSelect = (item) => {
+    setSelectedItem(item)
+    handleInputChange("dancestyles", item.name)
+  }
+  const onRoleSelect = (item) => {
+    setSelectedRole(item)
+    handleInputChange("role", item.name)
+  }
+
+    console.log(values);
```
Comment (MartinSWDev) [inline src/components/CreateProfile.jsx:79]: console.log needs removing

**Review by MartinSWDev — APPROVED** [review]

**Review by BrokenRecord85 — COMMENTED** [review]
As above

**Review by BrokenRecord85 — APPROVED** [review]

**Review by h-chagas — APPROVED** [review]
Please see comments from Martin above

**Pull request merged**